src/amplify-migration/amplifyMigrationScript.ts

```diff
@@ -1,30 +1,35 @@
-const simpleGit = require("simple-git");
```

Comment:
In the PR description, you talked about fixing bugs 1 - 3 and partially solving bug 4. How about bug 5 with JSDOM replacing the special chars. Is that also fixed in the PR?

Reply:
I have placed a comment below on an example of a place that it is fixed in, let me know if that is clear enough